common/pulp/common/error_codes.py

```diff
@@ -96,3 +96,4 @@
                              " directory:  %(path)s"), ['path'])
 PLP1008 = Error("PLP1008", _("The importer type %(importer_type_id)s does not exist"),
                 ['importer_type_id'])
+PLP1009 = Error("PLP1009", _("Request data is not valid JSON"), [])
```

Comment:
data is kind of an ambiguous term in HTTP land.

s/Request data is not valid JSON/The request body does not contain valid JSON/